packages/config/src/utils/normalize.js

```diff
@@ -7,30 +7,26 @@ const CAMELCASE_MAP = new Map([
   ['javascript', 'JavaScript']
 ]);
 
-// Converts kebab-cased and snake_cased strings to camelCase.
-const KEBAB_SNAKE_REG = /[-_]([^-_]+)/g;
+// Regular expression that matches words from boundaries or consecutive casing
+const WORD_REG = /[a-z]{2,}|[A-Z]{2,}|[0-9]{2,}|[^-_\s]+?(?=[A-Z0-9-_\s]|$)/g;
```

Comment:
And the regex grows 

![oh my](https://c.tenor.com/CMBmGu9CmUEAAAAC/oh-my-shocked.gif)